libs/stokesian_dynamics/thrust_wrapper.hpp

```diff
@@ -0,0 +1,192 @@
+/** \file
+ *  This file provides a wrapper around THRUST functions and types. In case
+ *  that CUDA/THRUST is not present, equivalent standard C++ types and 
+ *  functions are used.
+ */
+
+#pragma once
+
+
+
+// Dependencies with THRUST
+#ifdef STOKESIAN_DYNAMICS_GPU
```

Comment:
@hmenke Is this what you mean? In its current form, Thrust is indeed only used when the GPU is used. We should change this condition so that Thrust can also be used to do parallel CPU computations.

Reply:
Carl, please focus on the non-thrust/thrust_wrapper issues, for now.

The question of dependencies has to be sorted out with the core team, since any choice there results in us committing to support it in the CI and build infrastructure for the forseable future.